test/date_types.test.js

```diff
@@ -202,6 +217,18 @@ describe('MySQL DATE, DATETTIME, TIMESTAMP types on server with non local TZ (+0
     });
   });
 
+  it('should find model instance by date field passing string', function(done) {
```

Comment:
Same reason here. Test is failing locally for this. Why is this a valid test case to test for a `dateField` of type string when it expects a Date object?